blocks/library/table/table-block.js

```diff
@@ -89,6 +89,7 @@ export default class TableBlock extends wp.element.Component {
 						<li>
 							<DropdownMenu
 								icon="editor-table"
+								label={ wp.i18n.__( 'Edit Table' ) }
```

Comment:
Per #1332 (#1205), we should be explicit about dependencies on the i18n module, i.e. use `import { __ } from 'i18n';` instead of assuming global.